third_party/tonic/dart_state.h

```diff
@@ -49,6 +49,11 @@ class DartState : public std::enable_shared_from_this<DartState> {
   Dart_Isolate isolate() { return isolate_; }
   void SetIsolate(Dart_Isolate isolate);
 
+  // TODO(dnfield): delete this https://github.com/flutter/flutter/issues/50997
```

Comment:
Nit: `TODO(issue name or URL): Description of why this is present and when to remove it.`. Assignments are fluid and we can get those from the attached issue.

Reply:
Style guide still says to use this format: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions

Last time this came up, there wasn't much consensus around how to best handle it :\

Reply:
We follow https://google.github.io/styleguide/cppguide.html#TODO_Comments for C++. Since you added both the issue and your name, I felt the third format more relevant as your ldap was redundant if the issue was linked and the comment could have done with more context on when to remove.

Reply:
Alright, will update.